docs/reference/search/search-your-data/knn-search.asciidoc

```diff
@@ -156,13 +152,29 @@ most similar results from each shard. The search then merges the results from
 each shard to return the global top `k` nearest neighbors.
 
 You can increase `num_candidates` for more accurate results at the cost of
-slower search speeds. A search with a high number of `num_candidates` considers
-more candidates from each shard. This takes more time, but the search has a
-higher probability of finding the true `k` top nearest neighbors.
+slower search speeds. A search with a high value for `num_candidates`
+considers more candidates from each shard. This takes more time, but the
+search has a higher probability of finding the true `k` top nearest neighbors.
 
 Similarly, you can decrease `num_candidates` for faster searches with
 potentially less accurate results.
 
+[discrete]
+[[knn-indexing-considerations]]
+==== Indexing considerations
+
+Indexing vectors for approximate kNN search can take substantial time because
+of how expensive it is to build the ANN index structures. You may need to
```

Comment:
maybe, provide a definition in brackets what is ANN (approximate NN)?

Reply:
Good point, I'll use a different phrase since we don't use the acronym ANN in this reference.